.prow.sh

```diff
@@ -1,4 +1,4 @@
 #! /bin/bash
-
+export CSI_PROW_E2E_VERSION="release-1.31"
```

Comment:
Can you please comment why is this necessary and when we should remove it?

Reply:
added